modules/features/falcon_feature_commerce/falcon_feature_commerce.info.yml

```diff
@@ -12,13 +12,16 @@ dependencies:
   - commerce_product
   - commerce_store
   - falcon_commerce
+  - falcon_feature_appeal
```

Comment:
This raises a flag... if we want to enable Commerce + Gifts on Falcon, then we must enable Appeals too even if we don't need them. Do you have any ideas how to avoid this dependency?

Reply:
@duozersk promised to think about it 🥇

Reply:
Introduced new module and feature for donations.